_rest_routes.inc.php

```diff
@@ -94,6 +94,11 @@
         $data = (array)(json_decode(file_get_contents("php://input")));
         return (new EncounterRestController())->post($pid, $data);
     },
+    "PUT /api/patient/:pid/encounter/:eid" => function ($pid,$eid) {
+        RestConfig::authorization_check("encounters", "auth_a");
+        $data = (array)(json_decode(file_get_contents("php://input")));
+        return (new EncounterRestController())->put($pid, $eid,$data);
```

Comment:
I think this would be easier to read the code.

```php
return (new EncounterRestController())->put($pid, $eid, $data);
```

Reply:
agreed, this breaks psr2 (ie. one of reasons travis is failing; should have space after commas in line 97 and 100). rec having your editor support psr2 (or getting the lint support in the docker where could fix all this with the one liner :) ). Note I generally don't stop PRs for this since I can fix this stuff quickly in subsequent commit.

Reply:
Done